2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_094126-aw2wsuw7/logs/debug.log
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_094126-aw2wsuw7/logs/debug-internal.log
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_init.py:_jupyter_setup():480] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f28ba204dc0>
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:41:26,896 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-10-28 09:41:26,896 INFO    MainThread:405 [wandb_init.py:init():646] re-initializing run, found existing run on stack: zguk9u0x
2024-10-28 09:41:26,897 INFO    MainThread:405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/zguk9u0x
2024-10-28 09:41:26,897 INFO    MainThread:405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-28 09:41:26,897 INFO    MainThread:405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:41:26,897 INFO    MainThread:405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:41:30,894 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:41:30,894 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:41:30,901 INFO    MainThread:405 [wandb_run.py:_footer_sync_info():3934] logging synced files
2024-10-28 09:41:30,905 INFO    MainThread:405 [wandb_init.py:init():671] starting backend
2024-10-28 09:41:30,905 INFO    MainThread:405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:41:30,907 INFO    MainThread:405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:41:30,907 INFO    MainThread:405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:41:30,917 INFO    MainThread:405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:41:30,926 INFO    MainThread:405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:41:30,927 INFO    MainThread:405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:41:31,350 INFO    MainThread:405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:41:31,518 INFO    MainThread:405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:41:31,519 INFO    MainThread:405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:41:31,519 INFO    MainThread:405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:41:31,519 INFO    MainThread:405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:41:31,519 INFO    MainThread:405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:41:31,520 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:41:31,520 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:48:32,414 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:48:32,416 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:48:32,416 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:49:54,204 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:49:54,206 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:49:54,206 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:49:56,682 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:49:56,684 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:49:56,684 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:49:59,748 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:49:59,750 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:49:59,750 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:50:05,975 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:50:06,590 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:50:06,590 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:50:32,912 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:50:32,914 INFO    MainThread:405 [pyagent.py:run():322] Starting sweep agent: entity=None, project=None, count=None
2024-10-28 09:50:33,549 WARNING MsgRouterThr:405 [router.py:message_loop():77] message_loop has been closed
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'resnet18', 'entity': 'gachon-university-fdslab', 'root_dir': '/home/jovyan', 'sweep_id': '0a8fjf3n', 'run_id': 'lq3r3oat', 'sweep_param_path': '/home/jovyan/wandb/sweep-0a8fjf3n/config-lq3r3oat.yaml'}
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_095035-lq3r3oat/logs/debug.log
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_095035-lq3r3oat/logs/debug-internal.log
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_init.py:init():628] wandb.init called with sweep_config: {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'sgd'}
config: {}
2024-10-28 09:50:35,762 INFO    Thread-14 :405 [wandb_init.py:init():671] starting backend
2024-10-28 09:50:35,763 INFO    Thread-14 :405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:50:35,767 INFO    Thread-14 :405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:50:35,767 INFO    Thread-14 :405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:50:35,782 INFO    Thread-14 :405 [wandb_run.py:_config_callback():1390] config_cb None None {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'sgd'}
2024-10-28 09:50:35,783 INFO    Thread-14 :405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:50:35,795 INFO    Thread-14 :405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:50:35,796 INFO    Thread-14 :405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:50:36,160 INFO    Thread-14 :405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:50:36,332 INFO    Thread-14 :405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:50:36,332 INFO    Thread-14 :405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:50:36,332 INFO    Thread-14 :405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:50:36,333 INFO    Thread-14 :405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:50:36,333 INFO    Thread-14 :405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:50:36,519 INFO    Thread-14 :405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/lq3r3oat
2024-10-28 09:50:36,519 INFO    Thread-14 :405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 1
2024-10-28 09:50:36,519 INFO    Thread-14 :405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:50:36,520 INFO    Thread-14 :405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:50:39,008 INFO    Thread-14 :405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:50:39,008 INFO    Thread-14 :405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:50:39,012 INFO    Thread-14 :405 [wandb_run.py:_footer_sync_info():3934] logging synced files
2024-10-28 09:50:39,017 ERROR   MainThread:405 [pyagent.py:_run_jobs_from_queue():234] Run lq3r3oat errored:
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/wandb_config.py", line 165, in __getattr__
    return self.__getitem__(key)
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/wandb_config.py", line 130, in __getitem__
    return self._items[key]
KeyError: 'early_stopping_patience'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/site-packages/wandb/agents/pyagent.py", line 306, in _run_job
    self._function()
  File "<ipython-input-84-2e22f6ebdfe6>", line 23, in train_model_sweep
    early_stopping_patience = config.early_stopping_patience  # 조기 종료 기준 에포크 수
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/wandb_config.py", line 167, in __getattr__
    raise AttributeError(
AttributeError: <class 'wandb.sdk.wandb_config.Config'> object has no attribute 'early_stopping_patience'

2024-10-28 09:50:49,706 INFO    Thread-17 :405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:50:49,706 INFO    Thread-17 :405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:50:49,706 INFO    Thread-17 :405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:50:49,706 INFO    Thread-17 :405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:50:49,706 INFO    Thread-17 :405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'resnet18', 'entity': 'gachon-university-fdslab', 'root_dir': '/home/jovyan', 'run_id': 'n8tz8qrc', 'sweep_param_path': '/home/jovyan/wandb/sweep-0a8fjf3n/config-n8tz8qrc.yaml', 'sweep_id': '0a8fjf3n'}
2024-10-28 09:50:49,706 INFO    Thread-17 :405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:50:49,706 INFO    Thread-17 :405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:50:49,707 INFO    Thread-17 :405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:50:49,707 INFO    Thread-17 :405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_095049-n8tz8qrc/logs/debug.log
2024-10-28 09:50:49,707 INFO    Thread-17 :405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_095049-n8tz8qrc/logs/debug-internal.log
2024-10-28 09:50:49,707 INFO    Thread-17 :405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:50:49,707 INFO    Thread-17 :405 [wandb_init.py:init():628] wandb.init called with sweep_config: {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
config: {}
2024-10-28 09:50:49,707 INFO    Thread-17 :405 [wandb_init.py:init():671] starting backend
2024-10-28 09:50:49,707 INFO    Thread-17 :405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:50:49,711 INFO    Thread-17 :405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:50:49,712 INFO    Thread-17 :405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:50:49,725 INFO    Thread-17 :405 [wandb_run.py:_config_callback():1390] config_cb None None {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
2024-10-28 09:50:49,727 INFO    Thread-17 :405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:50:49,738 INFO    Thread-17 :405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:50:49,739 INFO    Thread-17 :405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:50:50,064 INFO    Thread-17 :405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:50:50,236 INFO    Thread-17 :405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:50:50,236 INFO    Thread-17 :405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:50:50,236 INFO    Thread-17 :405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:50:50,236 INFO    Thread-17 :405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:50:50,237 INFO    Thread-17 :405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:50:50,426 INFO    Thread-17 :405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/n8tz8qrc
2024-10-28 09:50:50,426 INFO    Thread-17 :405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 1
2024-10-28 09:50:50,426 INFO    Thread-17 :405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:50:50,426 INFO    Thread-17 :405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:50:52,053 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:50:52,054 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:50:52,730 INFO    Thread-17 :405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:50:52,730 INFO    Thread-17 :405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:50:52,730 ERROR   Thread-17 :405 [wandb_run.py:_atexit_cleanup():2447] Problem finishing run
Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/wandb_config.py", line 165, in __getattr__
    return self.__getitem__(key)
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/wandb_config.py", line 130, in __getitem__
    return self._items[key]
KeyError: 'early_stopping_patience'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<ipython-input-84-2e22f6ebdfe6>", line 23, in train_model_sweep
    early_stopping_patience = config.early_stopping_patience  # 조기 종료 기준 에포크 수
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/wandb_config.py", line 167, in __getattr__
    raise AttributeError(
AttributeError: <class 'wandb.sdk.wandb_config.Config'> object has no attribute 'early_stopping_patience'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 2438, in _atexit_cleanup
    self._on_finish()
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 2727, in _on_finish
    self._backend.cleanup()
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/backend/backend.py", line 214, in cleanup
    self.interface.join()
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/interface/interface_shared.py", line 537, in join
    self._router.join()
  File "/opt/conda/lib/python3.8/site-packages/wandb/sdk/interface/router.py", line 100, in join
    self._thread.join()
  File "/opt/conda/lib/python3.8/threading.py", line 1011, in join
    self._wait_for_tstate_lock()
  File "/opt/conda/lib/python3.8/threading.py", line 1027, in _wait_for_tstate_lock
    elif lock.acquire(block, timeout):
Exception
2024-10-28 09:51:53,443 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:51:53,507 INFO    MainThread:405 [wandb_settings.py:_apply_login():1972] Applying login settings: {}
2024-10-28 09:51:53,953 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:51:53,954 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:51:59,821 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:51:59,866 INFO    MainThread:405 [wandb_settings.py:_apply_login():1972] Applying login settings: {}
2024-10-28 09:52:00,325 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:52:00,325 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:52:26,511 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:52:26,565 INFO    MainThread:405 [wandb_settings.py:_apply_login():1972] Applying login settings: {}
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'resnet18', 'entity': 'gachon-university-fdslab', 'root_dir': '/home/jovyan', 'run_id': 'n8tz8qrc', 'sweep_param_path': '/home/jovyan/wandb/sweep-0a8fjf3n/config-n8tz8qrc.yaml', 'sweep_id': '0a8fjf3n'}
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_095226-n8tz8qrc/logs/debug.log
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_095226-n8tz8qrc/logs/debug-internal.log
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:52:26,570 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
config: {}
2024-10-28 09:52:26,571 INFO    MainThread:405 [wandb_init.py:init():671] starting backend
2024-10-28 09:52:26,571 INFO    MainThread:405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:52:26,575 INFO    MainThread:405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:52:26,575 INFO    MainThread:405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:52:26,588 INFO    MainThread:405 [wandb_run.py:_config_callback():1390] config_cb None None {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
2024-10-28 09:52:26,589 INFO    MainThread:405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:52:26,600 INFO    MainThread:405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:52:26,601 INFO    MainThread:405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:52:26,931 INFO    MainThread:405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:52:27,104 INFO    MainThread:405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:52:27,104 INFO    MainThread:405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:52:27,104 INFO    MainThread:405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:52:27,105 INFO    MainThread:405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:52:27,105 INFO    MainThread:405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:52:27,106 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:52:27,106 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:52:29,230 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:52:29,285 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:52:29,285 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:52:29,285 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:52:29,285 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:52:29,285 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'resnet18', 'entity': 'gachon-university-fdslab', 'root_dir': '/home/jovyan', 'run_id': 'n8tz8qrc', 'sweep_param_path': '/home/jovyan/wandb/sweep-0a8fjf3n/config-n8tz8qrc.yaml', 'sweep_id': '0a8fjf3n'}
2024-10-28 09:52:29,285 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:52:29,285 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:52:29,285 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:52:29,286 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_095229-n8tz8qrc/logs/debug.log
2024-10-28 09:52:29,286 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_095229-n8tz8qrc/logs/debug-internal.log
2024-10-28 09:52:29,286 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:52:29,286 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
config: {}
2024-10-28 09:52:29,286 INFO    MainThread:405 [wandb_init.py:init():646] re-initializing run, found existing run on stack: n8tz8qrc
2024-10-28 09:52:29,287 INFO    MainThread:405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/n8tz8qrc
2024-10-28 09:52:29,287 INFO    MainThread:405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-28 09:52:29,288 INFO    MainThread:405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:52:29,288 INFO    MainThread:405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:52:31,526 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:52:31,526 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:52:31,530 INFO    MainThread:405 [wandb_run.py:_footer_sync_info():3934] logging synced files
2024-10-28 09:52:31,533 INFO    MainThread:405 [wandb_init.py:init():671] starting backend
2024-10-28 09:52:31,534 INFO    MainThread:405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:52:31,538 INFO    MainThread:405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:52:31,538 INFO    MainThread:405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:52:31,550 INFO    MainThread:405 [wandb_run.py:_config_callback():1390] config_cb None None {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
2024-10-28 09:52:31,551 INFO    MainThread:405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:52:31,560 INFO    MainThread:405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:52:31,560 INFO    MainThread:405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:52:31,927 INFO    MainThread:405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:52:32,100 INFO    MainThread:405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:52:32,100 INFO    MainThread:405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:52:32,101 INFO    MainThread:405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:52:32,101 INFO    MainThread:405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:52:32,102 INFO    MainThread:405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:52:32,102 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:52:32,103 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:52:42,794 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'resnet18', 'entity': 'gachon-university-fdslab', 'root_dir': '/home/jovyan', 'run_id': 'n8tz8qrc', 'sweep_param_path': '/home/jovyan/wandb/sweep-0a8fjf3n/config-n8tz8qrc.yaml', 'sweep_id': '0a8fjf3n'}
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_095242-n8tz8qrc/logs/debug.log
2024-10-28 09:52:42,852 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_095242-n8tz8qrc/logs/debug-internal.log
2024-10-28 09:52:42,853 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:52:42,853 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
config: {}
2024-10-28 09:52:42,853 INFO    MainThread:405 [wandb_init.py:init():646] re-initializing run, found existing run on stack: n8tz8qrc
2024-10-28 09:52:42,854 INFO    MainThread:405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/n8tz8qrc
2024-10-28 09:52:42,854 INFO    MainThread:405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-28 09:52:42,854 INFO    MainThread:405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:52:42,854 INFO    MainThread:405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:52:45,089 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:52:45,089 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:52:45,093 INFO    MainThread:405 [wandb_run.py:_footer_sync_info():3934] logging synced files
2024-10-28 09:52:45,097 INFO    MainThread:405 [wandb_init.py:init():671] starting backend
2024-10-28 09:52:45,097 INFO    MainThread:405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:52:45,102 INFO    MainThread:405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:52:45,102 INFO    MainThread:405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:52:45,115 INFO    MainThread:405 [wandb_run.py:_config_callback():1390] config_cb None None {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
2024-10-28 09:52:45,116 INFO    MainThread:405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:52:45,125 INFO    MainThread:405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:52:45,126 INFO    MainThread:405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:52:45,469 INFO    MainThread:405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:52:45,641 INFO    MainThread:405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:52:45,642 INFO    MainThread:405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:52:45,642 INFO    MainThread:405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:52:45,642 INFO    MainThread:405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:52:45,643 INFO    MainThread:405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:52:45,643 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:52:45,643 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:53:35,762 INFO    MainThread:405 [wandb_init.py:_resume_backend():450] resuming backend
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {'project': 'resnet18', 'entity': 'gachon-university-fdslab', 'root_dir': '/home/jovyan', 'run_id': 'n8tz8qrc', 'sweep_param_path': '/home/jovyan/wandb/sweep-0a8fjf3n/config-n8tz8qrc.yaml', 'sweep_id': '0a8fjf3n'}
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:53:35,820 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_095335-n8tz8qrc/logs/debug.log
2024-10-28 09:53:35,821 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_095335-n8tz8qrc/logs/debug-internal.log
2024-10-28 09:53:35,821 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:53:35,821 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
config: {}
2024-10-28 09:53:35,821 INFO    MainThread:405 [wandb_init.py:init():646] re-initializing run, found existing run on stack: n8tz8qrc
2024-10-28 09:53:35,822 INFO    MainThread:405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/n8tz8qrc
2024-10-28 09:53:35,823 INFO    MainThread:405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-28 09:53:35,823 INFO    MainThread:405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:53:35,823 INFO    MainThread:405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:53:38,061 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:53:38,061 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:53:38,065 INFO    MainThread:405 [wandb_run.py:_footer_sync_info():3934] logging synced files
2024-10-28 09:53:38,071 INFO    MainThread:405 [wandb_init.py:init():671] starting backend
2024-10-28 09:53:38,071 INFO    MainThread:405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:53:38,076 INFO    MainThread:405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:53:38,076 INFO    MainThread:405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:53:38,091 INFO    MainThread:405 [wandb_run.py:_config_callback():1390] config_cb None None {'batch_size': 16, 'epochs': 50, 'learning_rate': 0.01, 'optimizer': 'rmsprop'}
2024-10-28 09:53:38,091 INFO    MainThread:405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:53:38,101 INFO    MainThread:405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:53:38,101 INFO    MainThread:405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:53:38,402 INFO    MainThread:405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:53:38,570 INFO    MainThread:405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:53:38,571 INFO    MainThread:405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:53:38,571 INFO    MainThread:405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:53:38,571 INFO    MainThread:405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:53:38,571 INFO    MainThread:405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:53:38,572 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:53:38,572 INFO    MainThread:405 [wandb_init.py:_pause_backend():445] pausing backend
2024-10-28 09:53:46,637 WARNING MsgRouterThr:405 [router.py:message_loop():77] message_loop has been closed
